src/sql/util.py

```diff
@@ -185,3 +185,39 @@ def _is_table_exists(table: str, with_: str, conn) -> bool:
                 pass
 
     return False
+
+
+def flatten(src, ltypes=(list, tuple)):
```

Comment:
I think this function can be quite handy. Can you add some tests for it? (test_util.py)

Reply:
Sounds good, added [15423a9](https://github.com/ploomber/jupysql/pull/152/commits/15423a9fbea99a88f297c6aac3635cb4011e55f5)